src/Adapter/MSTest.CoreAdapter/ObjectModel/TestMethod.cs

```diff
@@ -49,6 +50,11 @@ public TestMethod(string name, string fullClassName, string assemblyName,  bool
         /// </summary>
         public string Name { get; private set; }
 
+        /// <summary>
+        /// Gets the display name of the test method
+        /// </summary>
+        public string DisplayName { get; private set; }
```

Comment:
As **DisplayName** has a **public** access modifier I believe it should be reflected as well in the _interface_ **ITestMethod**.
That's the _interface_ **ITestMethod** should declare:

```
public interface ITestMethod
    {
        /// <summary>
        /// Gets the display name of the test method
        /// </summary>
        string DisplayName { get; private set; }

       // The rest of the properties...
}
```

Reply:
I think you are right. While not ever public member on a class needs  to be part of it's interface, this seems to make a lot of sense. 🙂

Reply:
> I think you are right. While not ever public member on a class needs to be part of it's interface, this seems to make a lot of sense. 🙂

Yes, agree on that but I think it could be useful if the interface can offer as well this **DisplayName** .
So, how can I contribute (with code) to this PR? I have  already cloned @ahmedalejo fork just in case ;)

Reply:
@sebainones your are correct, that should be it.

Reply:
@ahmedalejo please let me know how could I help you (contribute) with this. 
If you are opening a new PR just tag me so I can review it or if I can code something even better ;)